.homeycompose/app.json

```diff
@@ -4,35 +4,72 @@
   "brandColor": "#ed0c13",
   "name": {
     "en": "MELCloud",
-    "fr": "MELCloud"
+    "fr": "MELCloud",
+    "nl": "MELCloud",
+    "sv": "MELCloud",
+    "es": "MELCloud"
   },
   "description": {
     "en": "Take control of your heat pumps",
-    "fr": "Prenez le contrôle de vos pompes à chaleur"
+    "fr": "Prenez le contrôle de vos pompes à chaleur",
+    "nl": "Neem de controle over uw warmtepompen",
+    "sv": "Ta kontroll över dina värmepumpar",
```

Comment:
Most people only have one heat pump. Singular?

Reply:
It's less true for air-to-air than for air-to-water, and the goal is to sound like a slogan :-)